java/src/jmri/managers/DefaultInstanceInitializer.java

```diff
@@ -56,6 +56,10 @@
     @Override
     public <T> Object getDefault(Class<T> type) {
 
+        // In order for getDefault() to be called for a particular manager,
```

Comment:
Not sure what "to be called" means here.  

Maybe "for getDefault() to create a new object"?

Reply:
Thanks. I have updated the text.